src/renderers/shared/stack/reconciler/ReactCompositeComponent.js

```diff
@@ -245,7 +245,8 @@ var ReactCompositeComponentMixin = {
     inst.props = publicProps;
     inst.context = publicContext;
     inst.refs = emptyObject;
-    inst.updater = ReactUpdateQueue;
+    inst.updater = transaction.getUpdateQueue ?
```

Comment:
Wasn't there a couple more places in this file where `ReactUpdateQueue` is used directly? Shouldn't we take it from transaction in all those places?